src/frontend/ai-reviewer-admin/src/App.js

```diff
@@ -14,11 +15,26 @@ function App() {
 
   return (
     <>
-      <Header header={header} />
-      <Container className="content" >
-        <DocumentTypeEditor />
-      </Container>
-      <Footer />
+        <Router>
+        <div className="fixed-top header-div">
+          <Header header={header}/>
+        </div>
```

Comment:
I think in the frontend, the header/footer is outside of the router ... this was Alex's request back then.  Do you think it they should be included?  Just thinking that if someone access the site with no credentials, we may want to still show them the header/footer.

Reply:
The `Router` wouldn't affect whether a user can see the header or footer, It's only a wrapper component for the `Switch` and `Route` components

Reply:
My thinking was AuthenticationGuard would wrap the Container, this would give that effect. It won't break anything to move header and footer out of the Router, so we can, I was going by the `react-router-dom` documentation where they wrap the whole app in the Router component.

Reply:
Okay, sounds good. Yeah, using the frontend as an example is hardly the best thing.